dask_glm/tests/test_estimators.py

```diff
@@ -44,9 +44,14 @@ def test_pr_init(solver):
 
 
 @pytest.mark.parametrize('fit_intercept', [True, False])
-@pytest.mark.parametrize('is_sparse', [True, False])
-def test_fit(fit_intercept, is_sparse):
+@pytest.mark.parametrize('is_sparse,is_numpy', [
+                         (True, False),
+                         (False, False),
+                         (False, True)])
+def test_fit(fit_intercept, is_sparse, is_numpy):
     X, y = make_classification(n_samples=100, n_features=5, chunksize=10, is_sparse=is_sparse)
+    if is_numpy:
+        X, y = dask.compute(X, y)
     lr = LogisticRegression(fit_intercept=fit_intercept)
     lr.fit(X, y)
     lr.predict(X)
```

Comment:
I don't think I understand this test. When is `is_numpy` the case in a real-world example, IOW, will you ever have `X` and `y` be pure NumPy arrays that's worth testing with `LogisticRegression`? I assumed you'd only have Dask arrays (backed by Sparse or not).

Reply:
That's exactly what I tried to do, where both `X` and `y` are pure `numpy/cupy arrays`. Is that a feature we want? The current `dask-glm` only accepts `dask arrays`.

Reply:
I don't think that's a feature we need to support explicitly, I believe anybody using `dask-glm` would want to use Dask arrays rather than pure NumPy/CuPy ones.

Reply:
Thank you! I'll prioritize #89 then.